ProjectBank/Infrastructure.Tests/UserRepositoryTests.cs

```diff
@@ -0,0 +1,21 @@
+namespace ProjectBank.Infrastructure.Tests;
+public class UserRepositoryTests : TestSetup, IDisposable
+{
+    [Fact]
+    public void testExample()
+    {
+        var testUser = new User("Jeppe", "korg@itu.dk", "test", Role.Student);
```

Comment:
Personally i like unrelated fictitious names better as it seems more professional.

Reply:
Also the password parameter would probably be better suited with something like "password123" so the field is self-explanatory